src/cnab.rs

```diff
@@ -265,3 +278,19 @@ pub struct Destination {
     /// The fully qualified path to the destination file
     pub path: Option<PathBuf>,
 }
+
+/// A value that is produced by running an invocation image
+///
+/// Complies to CNAB Core 1.0
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct Output {
+    /// An optional exhaustive list of actions producing this output
+    pub apply_to: Option<Vec<String>>,
```

Comment:
The corresponding Go type is just a slice of strings, so could we get away without `Option` here and use an empty vector to represent "not present"?

Reply:
I don't have strong opinions about this one, but the Rust idiom seems to be to avoid allocating empty vectors if possible. I can go either way, though.

Reply:
Fair enough. Also it may be useful to distinguish between a provided empty list and the default.